core/src/main/scripts/import_data_validator/validateData.py

```diff
@@ -567,7 +566,10 @@ def validate(self):
                 return
 
             # parse the start-of-file comment lines
-            self.processTopLines(top_comments)
+            if not self.processTopLines(top_comments):
+                self.logger.info(
+                    'Invalid header comments, skipped data in file')
```

Comment:
what is meant by "skipped data"? Is the rest of the data not parsed because of this? If this is the case, it should be an error. Actually, in any case, this should be an error. If the header is invalid, the file should be considered invalid.